tests/base/src/test/java/org/wildfly/security/auth/FileSystemSecurityRealmTest.java

```diff
@@ -79,6 +92,26 @@
 // has dependency on wildfly-elytron-realm, wildfly-elytron-auth-server, wildfly-elytron-credential
 public class FileSystemSecurityRealmTest {
 
+    private static final Provider provider = WildFlyElytronPasswordProvider.getInstance();
+
+    @BeforeClass
+    public static void onBefore() throws Exception {
+        Security.addProvider(provider);
+    }
+
+    @AfterClass
+    public static void onAfter() throws Exception {
+        Security.removeProvider(provider.getName());
+    }
+    SecretKey secretKey = SecretKeyUtil.generateSecretKey(128);
+    KeyPairGenerator keyPairGen = KeyPairGenerator.getInstance("RSA");
+    KeyPair pair = keyPairGen.generateKeyPair();
+    PrivateKey privateKey = pair.getPrivate();
+    PublicKey publicKey = pair.getPublic();
```

Comment:
Very minor but these variable declarations could moved up under the provider declaration for better readability.

Reply:
They can also be private static.

Reply:
Moved up